miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppDownloader.kt

```diff
@@ -1,5 +1,6 @@
 package com.rakuten.tech.mobile.miniapp
 
+import android.util.Log
```

Comment:
@minh-rakuten I think the branch that you baselined it with is a different one.

Reply:
Oops, sorry 🙇 . The same baseline but forgot to remove logger codes.
Removed and totally checked.